src/journal-remote/journal-upload.c

```diff
@@ -580,6 +786,8 @@ static int parse_argv(int argc, char *argv[]) {
                 { "after-cursor", required_argument, NULL, ARG_AFTER_CURSOR   },
                 { "follow",       optional_argument, NULL, ARG_FOLLOW         },
                 { "save-state",   optional_argument, NULL, ARG_SAVE_STATE     },
+                { "compress",     optional_argument, NULL, ARG_COMPRESSION    },
+                { "header",       optional_argument, NULL, ARG_HEADER         },
```

Comment:
these two switches need a man page update too

Reply:
removed these cmd flags as they are not that useful